### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by josemigallas**
* Moves master ribbon down a bit (currently it's overlap by MastHead)

### CODE REVIEW

**Comment by thomasmaas** [comment]
Good catch on the ribbon, I never checked the master account!

File: app/assets/stylesheets/provider/_ribbon.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #ribbon-container {
   z-index: -1;
   position: absolute;
-  top: line-height-times(2);
+  top: line-height-times(2.5);
```
Comment (thomasmaas) [inline app/assets/stylesheets/provider/_ribbon.scss]: I prefer `line-height-times(5 / 2);` and let sass  do the actual divisions.

**Comment by thomasmaas** [comment]
I'd say rebase to just 1 commit :-)

**Comment by josemigallas** [comment]
I was hoping you would _Squash and merge_ it 😁

**Comment by thomasmaas** [comment]
https://github.com/3scale/porta/pull/31/commits/d970b0e356081e2ffdb3e79b1d8ea7d70d9dc718

**Pull request closed without merging**